lib/active_record/relation/query_methods_ext.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+module ActiveRecord
+  class Relation
+    module QueryMethodsExt
+      def from!(...) # :nodoc:
+        @force_index = nil
+        @index_hint = nil
+        super
+      end
+
+      # Set table index hint for the query to the
+      # given `index_name`, and `direction` (either
+      # `ASC` or `DESC`).
+      #
+      # Any call to `ActiveRecord::QueryMethods#from`
+      # will reset the index hint. Index hints are
+      # not set if the `from` clause is not a table
+      # name.
+      #
+      # @see https://www.cockroachlabs.com/docs/v22.2/table-expressions#force-index-selection
```

Comment:
I don't really like doing that as I saw that links to older versions of the doc are deleted. But I didn't find an alternative (for instance, `latest` instead of `v22.2`)

Reply:
good question! the info has moved here: https://www.cockroachlabs.com/docs/stable/select-clause.html#force-reverse-scan